docs/faq/end-users.md

```diff
@@ -1,49 +1,84 @@
 
-## What if the ERC-20 token I wish to transact with is not yet supported?
+It is as secure as it would be on the Layer 1, Ethereum blockchain. Hermez has completed 3 security audits before launching the mainnet to ensure network security.   
+For more information on Hermez security please visit, https://docs.hermez.io/#/about/security
 
-Most common tokens will be preloaded in the system. In any case, there is a rollup function for this. But you'll need to pay a small fee in order to add a token.
 
-## Which interface will users have available?
+# I'm getting an error message when I try to finalise my withdrawal, what's going on?
 
-Hermez will deliver a web wallet for users to operate with the network, and it will be connected to Metamask wallet for Ethereum key management.
+The most common reason for this error message is that there are not enough funds in your Ethereum L1 account to complete the withdrawal.  
 
-We expect integration with other wallets and independent implementations to appear.
+Withdrawals take place first on the Hermez L2, where the fee is paid in the same token as the transaction.  Next, it needs to move back to L1 which requires "gas" to cover the L1 transaction fee (this amount is determined by current prices on Ethereum Network. Hermez has no control over this fee).  If your L1 Ethereum account does not have enough funds to cover the gas charge, then the withdrawal cannot be completed and will produce the error message.  
 
-## Can coordinators take my money?
+To remedy this, please make sure you have enough funds in your L1 Ethereum account to cover the gas to complete the withdrawal.
 
-No.
+# Why is my Deposit taking so long to process?
 
-## Can coordinators censor transactions?
+Deposits move funds from Ethereum  L1 to L2 (Hermez) and must be forged by the Coordinator.  Depending on where the Coordinator is in the forging process, deposits may be as fast as 45 seconds or could take up to 15 minutes maximum. 
 
-Coordinators can decide which transactions they select to include in the batch to forge.
+# Can I use my Ledger/Trezor hardware with my Hermez Wallet?
 
-In case that a coordinator censures a transaction, users have the option to force coordinators to include their transaction. This is done by sending an on-chain transaction (`forceExit` or `forceTransfer`)
+Currently hardware wallets are not supported on Hermez, but should be supported in the near future. 
+
+# Fees and Tokens
+
+# Who makes money from the transaction fees?
+
+Transaction fees are collected by the Coordinator who placed the highest bid in the Coordinator auction.
+
+# How much will transactions cost on Hermez?
+
+Fees are used to pay for the infrastructure required to maintain the Hermez Coordinator plus the L1 costs involved in forging a new batch of transactions. Cost of the transaction will be split among the hundreds (or even thousands) of transactions in the batch, resulting in a lower cost compared to L1.
+
```

Comment:
Insert new question: ### How are transaction fees paid by users?

Transaction fees are paid in the same token of the transaction.